pull-request/src/app.js

```diff
@@ -7,7 +7,12 @@ var Person = function(first, last, middle) {
 Person.prototype = {
 
     whoAreYou : function() {
-        return this.first + (this.middle ? ' ' + this.middle: '') + ' ' + this.last;
+        if (this.first === NaN) {
```

Comment:
![Bug](https://sonarsource.github.io/sonar-github/bug.png 'Bug') Bug: Use a test of the format "a === a" instead. ([javascript:S2688](http://localhost:9740/coding_rules#rule_key=javascript%3AS2688))

[See it in SonarQube](http://localhost:9740/project/issues?id=training%3Apull-request&pullRequest=20&issues=AWe-3Wer5sewz8eX3TJO&open=AWe-3Wer5sewz8eX3TJO)

[ISSUE_KEY]: AWe-3Wer5sewz8eX3TJO